test/data_table_test.dart

```diff
@@ -394,6 +468,22 @@ class TableCellPO {
   PageLoaderElement materialCheckbox;
 }
 
+class TableHeaderCellPO {
+  @inject
+  PageLoader loader;
```

Comment:
this loader is unused

Reply:
I know its just copied from the other class, and its my fault to left there this unnecessary filed

Reply:
Fixed, but only for this class. Maybe need to do a future pass and check all unused properties.